src/content/developers/docs/nodes-and-clients/index.md

```diff
@@ -282,11 +282,19 @@ The most convenient and cheap way of running Ethereum node is to use a single bo
 
 Small, affordable and efficient devices like these are ideal for running a node at home.
 
-## Eth2 clients {#eth2-clients}
+## Consensus clients {#consensus-clients}
 
-There are new clients to support the [Eth2 upgrades](/upgrades/beacon-chain/). They will run the Beacon Chain and support the new [proof-of-stake](/developers/docs/consensus-mechanisms/pos/) consensus mechanism.
+There are new clients to support the [consensus upgrades](/upgrades/beacon-chain/). They will run the Beacon Chain and support the new [proof-of-stake](/developers/docs/consensus-mechanisms/pos/) consensus mechanism.
 
-[View Eth2 clients](/upgrades/get-involved/#clients).
+[View consensus clients](/upgrades/get-involved/#clients).
+
+| Client                                                      | Language   | Operating systems     | Networks                              | Sync strategies | State pruning |
```

Comment:
I would consider removing `Sync strategies` and `State pruning` columns here since these are just being developed in consensus clients, e.g. optimistic sync after the merge is not finished yet.